task-manager/src/components/CalendarComponent.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+    <v-col>
+        <v-sheet height="64">
+            <v-toolbar flat>
+                <v-btn
+                    outlined
+                    class="mr-4"
+                    color="grey darken-2"
+                    @click="setToday"
+                >
+                    Today
+                </v-btn>
+                <v-btn fab text small color="grey darken-2" @click="prev">
+                    <v-icon small> mdi-chevron-left </v-icon>
+                </v-btn>
+                <v-btn fab text small color="grey darken-2" @click="next">
+                    <v-icon small> mdi-chevron-right </v-icon>
+                </v-btn>
+                <v-toolbar-title v-if="$refs.calendar">
+                    {{ $refs.calendar.title }}
+                </v-toolbar-title>
+                <v-spacer></v-spacer>
+            </v-toolbar>
+        </v-sheet>
+        <v-sheet>
+            <v-calendar
+                ref="calendar"
+                v-model="focus"
+                color="primary"
+                elevation="2"
+                outlined
+                :first-day-of-week="1"
+                :events="events"
+            >
+            </v-calendar>
+        </v-sheet>
+    </v-col>
+</template>
+
+<script>
+import { mapGetters } from "vuex";
+
+export default {
+    name: "CalendarComponent",
+    data() {
+        return {
+            focus: "",
+        };
+    },
+    mounted() {
+        this.focus = Date.now();
+        this.$refs.calendar.checkChange();
+    },
+    computed: {
+        ...mapGetters(["getTasksByLoggedUser"]),
+        events() {
+            const eventData = this.getTasksByLoggedUser.reduce((prev, curr) => {
```

Comment:
can we use for each here and if statement?

Reply:
for each can be used, but here I am accumulating array values into one, that's why I have used reduce.